From 36ec9ed88c5c476f87a2eab40495311ff71cb02d Mon Sep 17 00:00:00 2001
From: Mike Snitzer <snitzer@redhat.com>
Date: Wed, 16 Jul 2014 20:42:05 -0400
Subject: [md] dm: make dm_table_alloc_md_mempools static

Message-id: <1405543364-20901-2-git-send-email-snitzer@redhat.com>
Patchwork-id: 85885
O-Subject: [RHEL7.1 PATCH 01/40] dm: make dm_table_alloc_md_mempools static
Bugzilla: 1117872
RH-Acked-by: Mikulas Patocka <mpatocka@redhat.com>
RH-Acked-by: Heinz Mauelshagen <heinzm@redhat.com>
RH-Acked-by: Joe Thornber <thornber@redhat.com>

BZ: 1117872

Upstream commit 473c36dfeecf4e49db928f3284b2fbe981f8c284
Author: Mikulas Patocka <mpatocka@redhat.com>
Date:   Thu Feb 13 13:43:32 2014 -0500

    dm: make dm_table_alloc_md_mempools static

    Make the function dm_table_alloc_md_mempools static because it is not
    called from another file.

    Signed-off-by: Mikulas Patocka <mpatocka@redhat.com>
    Signed-off-by: Mike Snitzer <snitzer@redhat.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/md/dm-table.c b/drivers/md/dm-table.c
index 6a7f2b8..2ae35b2 100644
--- a/drivers/md/dm-table.c
+++ b/drivers/md/dm-table.c
@@ -945,7 +945,7 @@ bool dm_table_request_based(struct dm_table *t)
  return dm_table_get_type(t) == DM_TYPE_REQUEST_BASED;
 }
 
-int dm_table_alloc_md_mempools(struct dm_table *t)
+static int dm_table_alloc_md_mempools(struct dm_table *t)
 {
  unsigned type = dm_table_get_type(t);
  unsigned per_bio_data_size = 0;
diff --git a/drivers/md/dm.h b/drivers/md/dm.h
index c4569f0..88cc58c 100644
--- a/drivers/md/dm.h
+++ b/drivers/md/dm.h
@@ -73,7 +73,6 @@ unsigned dm_table_get_type(struct dm_table *t);
 struct target_type *dm_table_get_immutable_target_type(struct dm_table *t);
 bool dm_table_request_based(struct dm_table *t);
 bool dm_table_supports_discards(struct dm_table *t);
-int dm_table_alloc_md_mempools(struct dm_table *t);
 void dm_table_free_md_mempools(struct dm_table *t);
 struct dm_md_mempools *dm_table_get_md_mempools(struct dm_table *t);
 
-- 
1.7.1